### PR TITLE
HYDRA-762 - Investigate why the tumbling performance of USD scene is slower than corresponding Maya Scene under Hydra Storm

### PR DESCRIPTION
**Pull request by vlasovi**
Fixing double-rendering of USD prims + autotest

### CODE REVIEW

**Comment by vlasovi** [comment]
FYI : @lilike-adsk @roopavr-adsk @debloip-adsk @lanierd-adsk

**Comment by lilike-adsk** [comment]
Nice catch! @vlasovi, I assume now, we can get similar fps on USD scene as corresponding Maya scene , right?

**Comment by vlasovi** [comment]
> Nice catch! @vlasovi, I assume now, we can get similar fps on USD scene as corresponding Maya scene , right?

Yes

File: lib/adskHydraSceneBrowser/test/adskHydraSceneBrowserTestFixture.cpp [hunk 1]
```diff
@@ -90,6 +91,60 @@ void AdskHydraSceneBrowserTestFixture::SetReferenceSceneIndex(
     sceneIndex = referenceSceneIndex;
 }
 
+void AdskHydraSceneBrowserTestFixture::VerifySceneCorrectness()
+{
+    // Traverse prim hierarchy
+    QTreeWidgetItemIterator itPrimsTreeWidget = GetIteratorForTree(_primHierarchyWidget);
```
Comment (ppt-adsk) [inline lib/adskHydraSceneBrowser/test/adskHydraSceneBrowserTestFixture.cpp]: Thanks for the test, looks good, but it feels out of place to tie it to the scene browser.  I asked @debloip-adsk and he agreed with me, saying that 
> with APIs like GetChildPrimPaths, GetPrim, Get methods for container data sources (or GetElement for vector data sources) + casts, and GetValue/GetTypeValue methods on data sources, the whole scene index scene hierarchy can be inspected.

Here's an example you might like:
https://github.com/Autodesk/maya-hydra/blob/dev/test/lib/mayaUsd/render/mayaToHydra/cpp/testPrimInstancing.cpp
It shouldn't change the logic of the test.

**Review by ppt-adsk — CHANGES_REQUESTED** [review]
Looks good to me, though I wouldn't tie the test to the scene browser.

**Review by ppt-adsk — APPROVED** [review]
Great work, thanks for the test!

**Pull request merged**